google/cloud/aiplatform/models.py

```diff
@@ -50,18 +57,22 @@ def __init__(
         location: Optional[str] = None,
         credentials: Optional[auth_credentials.Credentials] = None,
     ):
-        """Retrieves the model resource and instanties it's representation.
+        """Retrieves the model resource and instantiates it's representation.
 
         Args:
-            model_name (str): The name of the model to retrieve.
+            model_name (str):
+                Required. A fully-qualified model resource name or model ID.
+                Example: "projects/123/locations/us-central1/models/456" or
+                "456" when project and location are initialized or passed.
             project (str):
-                Optional project to retrieve model from. If not set, project set in
-                aiplatform.init will be used.
+                Optional project to retrieve model from. If not set, project
+                set in aiplatform.init will be used.
             location (str):
-                Optional location to retrieve model from. If not set, location set in
-                aiplatform.init will be used.
-            credentials (auth_credentials.Credentials):
-                Optional credentials to use to retrieve the model.
+                Optional location to retrieve model from. If not set, location
+                set in aiplatform.init will be used.
+            credentials: Optional[auth_credentials.Credentials]=None,
+                Custom credentials to use to upload this model. Overrides
```

Comment:
nit: Should this read `If not set, credentials set in aiplatform.init will be used.` for consistency?